vuepress/docs/next/docs/compose/preinstalled-widgets.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebarDepth: 2
+---
+
+# Preinstalled Widgets
+
+
+The Entando Platform includes a number of useful components to accelerate application development. These consist of widgets, page templates, content templates and content types. This page introduces the CMS, navigation, page, SEO and system widgets that are available out of the box. 
+
+## Widget Attributes
+
+All widgets are required to have the following attributes. These are mandatory regardless of whether a widget is preinstalled, user-created, or inside a bundle.
```

Comment:
do these requirements apply to all MFEs?

thought this page was only for pre-installed

Reply:
yes to both. this is a bit tricky because we don't have a page in docs dedicated to widgets in general. i put this info here rather than overload and break structure within the bundle component details page. these attributes are described for other entities and we need a place for this to land... the follow-up ticket where we add examples might generalize this page a bit more, which is another option (which i might prefer); we could rename this page more generally and then discuss preinstalled widgets as a subsection. for now we have kind of an interim solution